tests/agent/conftest.py

```
@@ -0,0 +1,37 @@
+    """根据 marker 设置测试模式。"""
+    if item.get_closest_marker(UNIT):
+        _run_mode_store["mode"] = "mock"
+    elif item.get_closest_marker(INTEGRATION):
+        _run_mode_store["mode"] = "real"
+    else:
+        _run_mode_store["mode"] = "auto"
+
+
+@pytest.fixture
+def agent_harness():
+    """返回 AgentTestHarness 实例，mode 由 marker 自动决定。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)